rules/aws_cloudtrail_rules/aws_console_login.yml

```diff
@@ -0,0 +1,19 @@
+AnalysisType: rule
+LogTypes:
+    - AWS.CloudTrail
+DisplayName: "AWS Console Login"
+RuleID: "AWS.Console.Login"
+Enabled: true
+CreateAlert: false
+Severity: Low
```

Comment:
Since we're not creating an alert, it makes sense to me that this ought to be `Info`. But I'm not sure, this will be our first signal only rule published so we have some leeway to pick the standards we want to set.

Reply:
If CreateAlert is false, is Severity even required?

Reply:
Severity is not required if create alert is false

Reply:
are you sure https://github.com/panther-labs/panther-analysis/actions/runs/8658191912/job/23741601311#step:6:6843 ?)